src/test/java/org/apache/maven/plugins/resources/filters/ItFilter.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.maven.plugins.resources.filters;
 
+import javax.inject.Named;
```

Comment:
just saying `javax.inject` is now legacy and deprecated in favour of `jakarta.inject`.
So ironically the PR title doesn't look correct.